src/daria/image/image.py

```diff
@@ -353,14 +360,98 @@ def copy(self) -> "Image":
         """
         Returns a copy of the image object.
         """
-        return Image(self.img, self.origo, self.width, self.height)
+        return Image(self.img, self.metadata, color_space=self.colorspace)
```

Comment:
Following my comments from above, I would rather go for removing `color_space` from the keywords, and instead move it into `metadata`. Shouldn't it anyhow be already represented in `metadata` at this stage?

Reply:
Oh, you are correct. This was just me confusing myself and working on the code at several places at once and changing my mind in the middle of it all. I actually don't think that the keyword argument is even read when it is written like this.